cmd/sync-tags/main.go

```diff
@@ -182,10 +183,16 @@ func main() {
 	// create or update tags from kTagCommits as local tags with the given prefix
 	createdTags := []string{}
 	for name, kh := range kTagCommits {
+		// temporarily publish only alpha tags
+		if !strings.Contains(name, "alpha") {
+			continue
+		}
+
 		bName := name
 		if *prefix != "" {
 			bName = *prefix + name[1:] // remove the v
 		}
+		semverTag := name[:1] + "0" + name[2:] // replace v1.x.y with v0.x.y
```

Comment:
at the very last, only do this swap to replace `v1.` with `v0.` (otherwise, we'd collapse v1.x.y and v2.x.y to the same tags)

```
semverTag = name
if strings.HasPrefix(name, "v1.") {
  semverTag = "v0." + strings.TrimPrefix(name, "v1.")
}
```

Reply:
also, make sure name is actually a valid semver tag before triggering the semver path

Reply:
Done. Also updated the results of the latest run to https://github.com/prowtest. PTAL.